looper/conductor.py

```diff
@@ -62,6 +62,29 @@ def write_sample_yaml(namespaces):
     sample.to_yaml(get_sample_yaml_path(namespaces), add_prj_ref=False)
     return {"sample": sample}
 
+def write_namespaces(namespaces):
+    """
+    Save all namespaces to YAML.
+
+    :param dict namespaces: variable namespaces dict
+    :return dict: sample namespace dict
+    """
+    import yaml
+    # my_namespaces = namespaces["sample"].to_dict()
+    path = get_sample_yaml_path(namespaces)
+    my_namespaces = {}
+    for namespace, values in namespaces.items():
+        if namespace == "sample":
+            values.to_yaml(path)
+            my_namespaces.update({str(namespace):yaml.load(open(path))})
+        else:
+            my_namespaces.update({str(namespace):values.to_dict()})
+    
+    # print(my_namespaces)
+    with open(path,'w') as yamlfile:
+        yaml.dump(my_namespaces, yamlfile)
```

Comment:
This would _overwrite_ the sample.yaml file. I don't think this was the goal here.